config/owasp/suppressions.xml

```diff
@@ -35,5 +35,14 @@
     <notes>False-positive. See CCD-3249</notes>
     <cve>CVE-2016-1000027</cve>
   </suppress>
+  
```

Comment:
@dinesh1patel - can you please clean up the false positive section as well?

Reply:
Done

Reply:
perfect